java/test/jmri/jmrix/dccpp/DCCppMessageTest.java

```diff
@@ -324,27 +324,27 @@ public void testMonitorStringAccessoryDecoderMsgActivateFalse() {
 
     @Test
     public void testMakesAndMonitors() {
-        msg = DCCppMessage.parseDCCppMessage("F 123 22 1");
+        msg = new DCCppMessage("F 123 22 1");
         Assert.assertEquals("Monitor string", "Function Cmd: CAB: 123, FUNC: 2, State: 1, (No Reply Expected)", msg.toMonitorString());
         msg = DCCppMessage.makeFunctionV2Message(123, 44, 1);
         Assert.assertEquals("Monitor string", "Function Cmd: CAB: 123, FUNC: 4, State: 1, (No Reply Expected)", msg.toMonitorString());
         msg = DCCppMessage.makeForgetCabMessage(1234);
         Assert.assertEquals("Monitor string", "Forget Cab: CAB: 1234, (No Reply Expected)", msg.toMonitorString());
-        msg = DCCppMessage.parseDCCppMessage("- 1234");
+        msg = new DCCppMessage("- 1234");
         Assert.assertEquals("Monitor string", "Forget Cab: CAB: 1234, (No Reply Expected)", msg.toMonitorString());
-        msg = DCCppMessage.parseDCCppMessage("-");
+        msg = new DCCppMessage("-");
         Assert.assertEquals("Monitor string", "Forget Cab: CAB: [ALL], (No Reply Expected)", msg.toMonitorString());
-        msg = DCCppMessage.parseDCCppMessage("- 12345");
+        msg = new DCCppMessage("- 12345");
         Assert.assertNull("null on invalid address", msg);
```

Comment:
Windows CI fails on this line with the error message:
`DCCppMessageTest.testMakesAndMonitors:338 null on invalid address expected null, but was:<- 12345>`

Note that `msg` will never be null here. Either the `DCCppMessage` constructor fails and throws an exception, which you don't catch here, or DCCppMessage constructor returns a valid object.

You have several more cases like this on the lines that follows.

Reply:
Since the method `DCCppMessage.parseDCCppMessage()` is removed, the easiest solution is to simply remove the lines that tests that method.

Reply:
I am not a JAVA expert but msg = new DCCppMessage("- 12345"); is in my valid. It should construct a DCCppMessage object from a valid string "- 12345".
```
    public DCCppMessage(String s) {
        setBinary(false);
        setRetries(_nRetries);
        setTimeout(DCCppMessageTimeout);
        myMessage = new StringBuilder(s); // yes, copy... or... maybe not.
        toStringCache = s;
        // gather bytes in result
        setRegex();
        _nDataChars = myMessage.length();
        _dataChars = new int[_nDataChars];
    }
```
So I'd say the test is at error here and the code is correct. What to do about this?

Harald.

Reply:
Sorry, was a little too fast. I now think I know what's going on. This should test a message that should fail but it never will. Will just remove the test and make a new pull.

Harald.